### DISCUSSION TITLE
LLM 開發工具很實用

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2268] [General]
最近在玩 google ai studio 的大型語言模型 我利用這個平台成功開發了一個模型導入的我的生物資訊分析軟體中 讓他變成了自帶教學助理的分析軟體 大家有知道什麼開發平台嗎

**Comment by Teddy111802518** [comment]
目前我是只能做到把軟體的功能整理成問答資料集，拿去tune，不過我整理了30個問答，但是在測試的時候，問他跟我自己特定軟體的問題，還是回答得很不明確。

**Comment by Teddy111802518** [comment]
至於其他開發平台，我是知道nvidia開發的BioNemo ，他是一個把眾多模型彙整起來的大型操作平台，包含藥物開發的各種過程，大家熟知的alphafold2也在其中，並且提供雲端算力，據說超貴

**Comment by BOSENCHANG2004** [comment]
如果有同學對語音生成有興趣可以參考一個工具:GPT-SoVITS
能夠用自己的聲音去訓練，然後就可以輸入文本請模型輸出mp3
使用過效果非常好，就是對電腦顯示卡需求比較有要求。
https://github.com/RVC-Boss/GPT-SoVITS.git
